Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+
+public class UIManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+
+ 
+    public TextMeshProUGUI playerOneScoreLabel;
+    public TextMeshProUGUI playerOneLivesLeftLabel;
+    public GameObject playerOneDurability;
+    public GameObject playerOneHealth;
+    public GameObject playerOneInventory;
+
+
+    public int playerOneScoreNum;
+    public int playerOneLivesLeftNum;
+    public float playerOneDurabilityScore = 0.2f;
+    public float playerOneHealthScore;
+    public string playerOneInventoryType;
+
+    //public GameObject playerTwoScore;
+    //public GameObject playerTwoDurability;
+    //public GameObject playerTwoHealth;
+    //public GameObject playerTwoInventory;
+    //public int playerTwoScoreNum;
+    //public int playerTwoDurabilityScore;
+    //public int playerTwoHealthScore;
+    //public string playerTwoInventoryType;
+
+
+
+    private void Update()
+    {
+        if (Input.GetKeyDown("z"))
+        {
+            changePlayerOneScore(1);
+            Debug.Log("click");
+        }
+
+        if (Input.GetKeyDown("x"))
+        {
+            changePlayerOneDurability(0.4f);
+            Debug.Log("click");
+        }
+
+        if (Input.GetKeyDown("c"))
+        {
+            changePlayerOneHealth(0.4f);
+            Debug.Log("click");
+        }
+
+        if (Input.GetKeyDown("v"))
+        {
+            changePlayerOneLivesLeft(-1);
+        }
+
+    }
+
+
+    public void changePlayerOneScore(int changeBy)
```

Comment:
Again, as mentioned, these functions should be in their own class.
Also, the function name is not matching the naming scheme, please fix them all.
Please also take your naming seriously, the name `delta` might be a better naming for the parameter.